.changes/next-release/enhancement-RequestHeader-82118.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "enhancement",
+  "category": "Request Header",
```

Comment:
This may be better to put under `Lambda` since it's service specific, "Request Header" is a little nebulous. We'll want to capitalize Lambda in the description too.